frontend_tests/zjsunit/namespace.js

```diff
@@ -16,15 +17,22 @@ exports.patch_builtin = function (name, val) {
     return val;
 };
 
-exports.zrequire = function (name, fn) {
+exports.zrequire = function (name, fn, stub = null) {
     if (fn === undefined) {
         fn = '../../static/js/' + name;
     } else if (/^generated\/|^js\/|^shared\/|^third\//.test(fn)) {
         // FIXME: Stealing part of the NPM namespace is confusing.
         fn = '../../static/' + fn;
     }
     delete require.cache[require.resolve(fn)];
-    const obj = require(fn);
+    var obj;
+    if (!stub) {
+        obj = require(fn);
+    } else {
+        // Overrides the methods of the module with the one specified in stub.
+        // Non overridden methods of the module behaves like the original.
```

Comment:
That’s not true with `.noCallThru()`, right?

Reply:
Yeah. You are right. `noCallThru` alters the default behavior. It was not really required.